tools/src/main/scala/com/nvidia/spark/rapids/tool/planparser/SQLPlanParser.scala

```diff
@@ -196,9 +198,19 @@ object SQLPlanParser extends Logging {
       }
       val stagesInNode = getStagesInSQLNode(node, app)
       val supported = execInfos.isSupported && !ds && !containsUDF
+      // add Exec to the unsupported list if the speedup factor is 1 and all underlying
+      // expressions are supported.
+      val unSupportedExecs = if (!execInfos.isSupported && execInfos.speedupFactor == 1
```

Comment:
why do we need to check speedupFactor and nsupportedExprs checks?  Doesn't the isSupported for each exec cover this?

Reply:
You are right. Thanks for catching this.